src/backend/parts/data/URLCrawler.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+
+import {
+	Button,
+	TextControl,
+	Panel,
+	PanelRow
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies.
+ */
+import PostsTable from '../PostsTable';
+import UpsellContainer from '../UpsellContainer';
+
+const posts = [
+	{
+		id: 1,
+		title: 'https://example.com/page1'
+	},
+	{
+		id: 2,
+		title: 'https://example.com/page2'
+	},
+	{
+		id: 3,
+		title: 'https://example.com/page3'
+	},
+	{
+		id: 4,
+		title: 'https://example.com/page4'
+	},
+	{
+		id: 5,
+		title: 'https://example.com/page5'
+	}
+];
```

Comment:
Here, we can improve the name and generation like:

```js
const defaultExamplePosts = Array.from({ length: 5 }, (_, i) => ({ id: i + 1, title: `https://example.com/page${i + 1}` }));
```

Low line count and explicit usage in the name